Slim/Player/ReplayGain.pm

```diff
@@ -198,7 +198,7 @@ sub trackSampleRateMatch {
 	my $offset = shift;
 
 	my ($current_track, $compare_track) = $class->findTracksByIndex($client, $offset);
-	return 1 if (!$current_track || !$compare_track);
+	return if (!$current_track || !$compare_track);
```

Comment:
Are you reverting this change because we should no longer hit this check when in single track mode?

Reply:
Yes, because I assume it's fair to say that, regardless of use of smart crossfade or not, we don't want a track to intermix crossfade with itself. And because the test only kicks-in when crossfading is intermix (not fade-in/out) it feels right. 
Having said that, it's questionnable that sampleRateMatch() returns false when there is only one track, because stricly speaking it *does* match. Maybe better to add a test that disable self-intermix when there is onyl one track